src/main/kotlin/minesweeper/domain/Board.kt

```diff
@@ -0,0 +1,31 @@
+package minesweeper.domain
+
+internal class Board private constructor(val cellRows: List<CellRow>) {
+
+    companion object {
+        internal fun createBoard(boardSpec: BoardSpec, minePositions: (BoardSpec) -> List<Position> = this::randomMinePositions): Board {
+            val positions = minePositions(boardSpec)
+            val cellRows = mutableListOf<CellRow>()
+
+            repeat(boardSpec.height.value) {
+                val curY = it
+                val cells = (0 until boardSpec.width.value).map {
+                    val hasMine = positions.contains(Position(NaturalNumber(it), NaturalNumber(curY)))
+                    Cell(hasMine)
+                }
+                cellRows.add(CellRow(cells))
+            }
```

Comment:
`x`, `y`로 이름을 지어주는 건 어떨까요?

```suggestion
            repeat(boardSpec.height.value) { y ->
                val cells = (0 until boardSpec.width.value).map { x ->
                    val hasMine = positions.contains(Position(NaturalNumber(x), NaturalNumber(y)))
                    Cell(hasMine)
                }
                cellRows.add(CellRow(cells))
            }
```